Common++/header/Logger.h

```diff
@@ -11,6 +11,33 @@
 #define LOG_MODULE UndefinedLogModule
 #endif
 
+// Use __FILE_NAME__ to avoid leaking complete full path
+#ifdef __FILE_NAME__
+#define PCAPPP_FILENAME __FILE_NAME__
+#else
+#define PCAPPP_FILENAME __FILE__
+#endif
+
+#define PCPP_LOG(level, message) do \
+	{ \
+		std::ostringstream* sstream = pcpp::Logger::getInstance().internalCreateLogStream(); \
+		(*sstream) << message; \
+		pcpp::Logger::getInstance().internalPrintLogMessage(sstream, level, PCAPPP_FILENAME, __FUNCTION__, __LINE__); \
+	} while(0)
+
+#define PCPP_LOG_DEBUG(message) do \
+	{ \
+		if (pcpp::Logger::getInstance().logsEnabled() && pcpp::Logger::getInstance().isDebugEnabled(LOG_MODULE)) \
+		{ \
+			PCPP_LOG(pcpp::Logger::Debug, message); \
+		} \
+	} while(0)
+
+#define PCPP_LOG_ERROR(message) do \
+	{ \
+		PCPP_LOG(pcpp::Logger::Error, message); \
+	} while (0)
+
```

Comment:
Why did you move these macros to the top of the file?

Reply:
I think it was not intuitive to put them in the namespace where they don't believe.

Reply:
ok 👍